pkg/controller/plan/vm_name_handler.go

```diff
@@ -33,7 +33,7 @@ func (r *KubeVirt) changeVmNameDNS1123(vmName string, vmNamespace string) (gener
 
 // changes VM name to match DNS1123 RFC convention.
 func changeVmName(currName string) string {
-	var underscoreExcluded = regexp.MustCompile("[_.]")
+	var underscoreExcluded = regexp.MustCompile("[_]")
```

Comment:
we also need to replace or remove  remove "." if it is in the start or end of the name

https://github.com/kubernetes/apimachinery/blob/master/pkg/util/validation/validation.go#L206

Reply:
hmm that's get tricky

Reply:
remove the "." if they are in the start of end ?